ui/src/components/Preview/index.jsx

```diff
@@ -1,45 +1,78 @@
+import classNames from 'classnames';
+import { string, func } from 'prop-types';
 import React, { Component } from 'react';
 import Markdown from '@mozilla-frontend-infra/components/Markdown';
-import ListItem from '@material-ui/core/ListItem';
 import Tab from '@material-ui/core/Tab';
 import Tabs from '@material-ui/core/Tabs';
 import TextField from '@material-ui/core/TextField';
+import { withStyles } from '@material-ui/core/styles';
+import { fade } from '@material-ui/core/styles/colorManipulator';
+import { THEME } from '../../utils/constants';
 
+@withStyles(theme => {
+  const borderColor =
+    theme.palette.type === 'light'
+      ? fade(THEME.BLACK, 0.23)
+      : fade(THEME.WHITE, 0.23);
+
+  return {
+    tab: {
+      flexGrow: 1,
+    },
+    tabContent: {
+      marginTop: theme.spacing.unit,
+      padding: theme.spacing.unit,
+    },
+    markdownContainer: {
+      borderColor,
+      borderWidth: 1,
+      borderRadius: theme.shape.borderRadius,
+      borderStyle: 'solid',
+    },
+  };
+})
 export default class Preview extends Component {
+  static propTypes = {
+    onValueChange: func,
+    value: string,
+    placeholder: string,
+  };
+
   state = {
-    value: 0,
+    tabValue: 0,
   };
 
   handleChange = (event, value) => {
-    this.setState({ value });
+    this.setState({ tabValue: value });
   };
 
   render() {
-    const { description, onDescriptionChange, placeholder } = this.props;
-    const { value } = this.state;
+    const { value, onValueChange, placeholder, classes } = this.props;
+    const { tabValue } = this.state;
 
     return (
-      <div>
-        <ListItem>
-          <Tabs value={value} onChange={this.handleChange}>
-            <Tab label="Description" />
-            <Tab label="Preview" />
-          </Tabs>
-        </ListItem>
-        <ListItem>
-          {value === 0 && (
+      <div className={classNames(classes.tab)}>
+        <Tabs value={tabValue} onChange={this.handleChange}>
+          <Tab label="Write" />
+          <Tab label="Preview" />
+        </Tabs>
+        <div
+          className={classNames(classes.tabContent, {
+            [classes.markdownContainer]: tabValue === 1,
+          })}>
+          {tabValue === 0 && (
             <TextField
-              name="description"
```

Comment:
If we render this component directly as `<Preview />`, I don't think we would be able to change the value of the component since it is controlled. Let's make it work when uncontrolled (i.e., `<Preview />`). This is similar to something like `<TextField />` which works out of the box without having to specify the props.